lib/migrate.rb

```diff
@@ -3,6 +3,17 @@
 # run all trunk migrations first, then all others.
 #
 
+# Helper method
+#
+def foreign_key_delete_cascade source_table, target_table, foreign_key
+  <<-SQL
+    ALTER TABLE #{source_table} DROP CONSTRAINT #{source_table}_#{foreign_key}_fkey; 
```

Comment:
We could probably also do this via Sequel, but after reading the docs it seemed that it's clearer this way.